synapse/common.py

```diff
@@ -1234,7 +1209,22 @@ def _patch_http_cookies():
         return
     http.cookies._unquote = v_cookies._unquote
 
+def _patch_tornado_json():
+    import synapse.lib.json as s_json
+
+    if hasattr(tornado.escape, 'json_encode'):
+        # This exists for a specific reason. See the following URL for explanation:
+        # https://github.com/tornadoweb/tornado/blob/master/tornado/escape.py#L83-L96
```

Comment:
A permalink or a release tag based link might be better since these line highlights could change over time